extensions/shared/components/block-nudge/index.jsx

```diff
@@ -17,7 +17,7 @@ export const BlockNudge = ( { autosaveAndRedirect, buttonLabel, href, icon, subt
 					href={ href } // Only for server-side rendering, since onClick doesn't work there.
 					onClick={ autosaveAndRedirect }
 					target="_top"
-					isDefault
+					isSecondary
```

Comment:
If we upgrade the components dependency and start using `isSecondary` here, are there other buttons that will need upgrading?

Reply:
I don't think so, but the following components are also in that package so might have changed:
`TextControl`, `PanelBody`, `SVG`, `Path`, `withFallbackStyles`, `createSlotFill`, `Polygon`,`IconButton`, `ClipboardButton`, `TextControl`.

According to the [changelog](https://github.com/WordPress/gutenberg/blob/b3f900c34d41c282db5a6d5a43f8eaf5f091f3d6/packages/components/CHANGELOG.md) we should be ok except for IconButton...

Reply:
I added a new PR for the dependency change: https://github.com/Automattic/jetpack/pull/14934

Reply:
Cool, shall we back out the dependency change here or wait to merge #14934?

Reply:
Done!